.buildkite/pipeline.yml

```diff
@@ -162,6 +162,18 @@ steps:
     agents:
       queue: "t2medium"
 
+  - label: ":book: check documentation"
+    command: ".buildkite/steps/test-documentation.sh"
+    <<: *timeout
+    depends_on: "runner-3_6"
+    plugins:
+      <<: *plugins
+      docker-compose#v3.2.0:
+        <<: *compose-config
+        run: runner-3_6
+    agents:
+      queue: "t2medium"
```

Comment:
Could this run potentially run successfully on the default queue (t2micro)?

Reply:
I suspect it might be able to, but it would likely require pulling the `runner-3_6` image. I've been trying to keep steps using those images on `t2medium` and `t2large` instances, to have more cache hits.

Reply:
Fair enough 👍